pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/sierramets/SierraMetsWorkPairMerger.scala

```diff
@@ -0,0 +1,66 @@
+package uk.ac.wellcome.platform.merger.rules.sierramets
+
+import uk.ac.wellcome.models.work.internal.{
+  DigitalLocation,
+  Identifiable,
+  IdentifiableRedirect,
+  Item,
+  TransformedBaseWork,
+  Unidentifiable,
+  UnidentifiedRedirectedWork,
+  UnidentifiedWork
+}
+import uk.ac.wellcome.platform.merger.model.MergedWork
+import uk.ac.wellcome.platform.merger.rules.{MergerRule, WorkPairMerger}
+
+/**
+  * A METS work contains information relating to the digitisation of a Sierra work.
+  * Currently that amounts to a digital location on the item which contains
+  * the url to the iiif-presentation API and the license associated with it.
+  *
+  * If we have a Sierra and a METS work each one with a single item,
+  * we merge the digital location of the METS work into the locations of the Serra work.
```

Comment:
```suggestion
  * we merge the digital location of the METS work into the locations of the Sierra work.
```

Reply:
Sorry, I fixed this straight in master after merging this: https://github.com/wellcometrust/catalogue/commit/e09e1179b2f736a07a1d44a9dad18e8d5430b16a

Thanks!